iothub/device/devdoc/architecture.md

```diff
@@ -84,63 +62,54 @@ A `WaitForTransportClosedAsync` request is made right after a successfull open.
 
 Handling disconnects is going to apply the configured `RetryStrategy` indefinitely to recover existing subscriptions (e.g. methods, twin, etc.)
 
-### ErrorDelegatingHandler
-
-`ErrorDelegatingHandler` performs error adaptation for non-Azure IoT Hub specific errors thrown by the transport layer (e.g. SocketException or AmqpException) to `IoTHubException` types. Ideally this code should be pushed within the respective handlers. The pipeline class is present for legacy reasons only.
-
-The handler is responsible with logging application-level exceptions. It is guaranteed that these exceptions will be available to applications either through custom `IRetryStrategy` implementations or by having them thrown by the public APIs.
+### RetryDelegatingHandler
 
-#### Exception handling
+`RetryDelegatingHandler` class is responsible with applying the operation retry policy.<br>
+__Important:__ No other pipeline component should attempt operation retry.<br>
```

Comment:
Why html?

```suggestion
`RetryDelegatingHandler` class is responsible with applying the operation retry policy.

__Important:__ No other pipeline component should attempt operation retry.

```

Reply:
What I focused on here was the text. I didn't focus on the formatting used since what we had was working.